src/app/screens/Config/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Modal, Text, TouchableHighlight, View } from 'react-native';
+import { TextInput } from 'react-native';
+
+import styles from './styles';
+
+class Config extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  state = {
+    modalVisible: false,
+    ssid: "",
+    password: ""
+  };
+
+  setModalVisible(visible) {
+    this.setState({ modalVisible: visible });
+  }
+
+  handleSsidChange = newSsid => {
+    this.setState({ ssid: newSsid });
+  }
+
+  handlePasswordChange = newPassword => {
+    this.setState({ password: newPassword });
+  }
+
+  render() {
+    return (
+      <View style={styles.margin}>
+        <Modal
+          animationType="slide"
+          transparent={false}
+          visible={this.state.modalVisible}
+        >
+          <View style={styles.margin}>
+            <View>
+              <TouchableHighlight
+                onPress={() => {
```

Comment:
Se evita usar function arrows `() => {}` directo en el render ya que por cada llamado del render se va a crear una nueva function arrow.
Para evitar esto, hace que la función pertenezca a la clase declarandola así:

```
toggleModalVisible = () => this.setState(prevState => ({ modalVisible: !prevState.modalVisible })
```

Reply:
Entonces desde el render solamente pasas la funcion por referencia como
`onPress={this.toggleModalVisible}`
Esto evita que crees nuevas instancias de function arrow cada render